python/ray/tests/test_advanced_5.py

```diff
@@ -165,7 +165,7 @@ def test_actor_distribution_balance(ray_start_cluster_enabled, args):
     ray.init(address=cluster.address)
     cluster.wait_for_nodes()
 
-    @ray.remote(memory=100 * 1024 ** 2, num_cpus=0.01)
+    @ray.remote(memory=100 * 1024 ** 2, num_cpus=0.01, scheduling_strategy="SPREAD")
```

Comment:
Previously, gcs actor scheduler aims spreading actors evenly, which is now equal to the `SPREAD` policy. If using the default hybrid policy, we can not guarantee balanced distribution, because tasks are stacked under the threshold.